### PR TITLE
OS-1: add REST API for Vh module

### PR DESCRIPTION
**Pull request by Maveric-k07**
## Issue that this pull request solves

Closes: #1209

## Types of changes

_Put an `x` in the boxes that apply_

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Other (please describe):

## Checklist

_Put an `x` in the boxes that apply_

-   [x] My code follows the [style guidelines of this project](https://docs.google.com/document/d/1GI2Hile8UbGZ82gFe1y4wAVPcNPXip1cmXTzog1S41U/edit)
-   [x] I have performed a self-review of my own code
-   [x] I have created new branch for this pull request
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [x] My changes generate no new warnings
-   [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

No change in User Interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Comment by ashutosh-2002** [comment]
Approved @akshatnema

File: FusionIIIT/applications/visitor_hostel/api/views.py [hunk 1]
```diff
@@ -0,0 +1,1044 @@
...
+                   'canceled_bookings': canceled_bookings,
+                   'dashboard_bookings' : dashboard_bookings,
+
+                   'bills': bills,
+                   # 'all_rooms_status' : all_rooms_status,
+                   'available_rooms': available_rooms,
+                   'forwarded_rooms': forwarded_rooms,
+                   # 'booked_rooms' : booked_rooms,
+                   # 'under_maintainence_rooms' : under_maintainence_rooms,
+                   # 'occupied_rooms' : occupied_rooms,
```
Comment (akshatnema) [inline FusionIIIT/applications/visitor_hostel/api/views.py]: Kindly remove these types of comments from the File Changes of PR.

File: FusionIIIT/applications/visitor_hostel/api/views.py [hunk 2]
```diff
@@ -0,0 +1,936 @@
...
+                                                                                                                      booking_to__lte=date2) | Q(booking_from__lte=date2, booking_to__gte=date2) | Q(booking_from__lte=date1, booking_to__gte=date1) | Q(booking_from__gte=date1, booking_to__lte=date2) | Q(booking_from__lte=date2, booking_to__gte=date2))
+    # bill_details = Bill.objects.filter(Q(booking__booking_from__lte=date1, booking__booking_to__gte=date1, booking__status="Confirmed") | Q(booking__booking_from__gte=date1,
+    #                                                                                                                   booking__booking_to__lte=date2, booking__status="Confirmed") | Q(booking__booking_from__lte=date2, booking__booking_to__gte=date2, status="Confirmed") | Q(booking_from__lte=date1, booking__booking_to__gte=date1, status="CheckedIn") | Q(booking__booking_from__gte=date1, booking__booking_to__lte=date2, booking__status="CheckedIn") | Q(booking__booking_from__lte=date2, booking__booking_to__gte=date2, booking__status="CheckedIn"))
```
Comment (akshatnema) [inline FusionIIIT/applications/visitor_hostel/api/views.py]:
```suggestion
```

**Review by akshatnema — CHANGES_REQUESTED** [review]

**Comment by akshatnema** [comment]
Kindly remove the same type of comments from this file FusionIIIT/applications/visitor_hostel/api/views.py

**Pull request merged**